app/views/events/show.html.erb

```diff
@@ -93,11 +93,12 @@
           <% @tickets.each do |ticket| %>
             <%if ticket.user %>
               <li>
-                <%= link_to(url_for_twitter(ticket.user)) do %>
-                  <%= image_tag ticket.user.image_url, width: 20, height: 20 %>
-                  <%= "@#{ticket.user.nickname}" %>
+                <% if ticket.user.provider == 'twitter' %>
```

Comment:
__[rails_best_practices]__
* law of demeter [:ref](http://rails-bestpractices.com/posts/2010/07/24/the-law-of-demeter/)